examples/blueprint-construct/index.ts

```diff
@@ -74,6 +75,19 @@ export default class BlueprintConstruct {
             new blueprints.addons.CoreDnsAddOn(),
             new blueprints.addons.KubeProxyAddOn(),
             new blueprints.addons.OpaGatekeeperAddOn(),
+            new blueprints.addons.AckAddOn({
+                skipVersionValidation: true
+            }),
+            new blueprints.addons.AckAddOn({
+                skipVersionValidation: true,
+                name: "rds-chart",
+                chart: "rds-chart",
```

Comment:
The blueprint should make it easy for the customers to use add-on. They don't need to know the oci repo and convention between AWS service name, chart name, repo, etc. See my other comment on the design of the addon props.

Reply:
This will make it very difficult to maintain it as ACK constantly adds one service after the other. Currently 47 services are supported today. Adding and maintaining is each month is something we dont want to do. Lets talk about it.